### ISSUE TITLE
Option to only fail on certain impact levels

### ISSUE THREAD

**Issue by eoinobrien** [chanzuckerberg/axe-storybook-testing#9]
[Axe has different impacts for different rules](https://github.com/dequelabs/axe-core/blob/develop/doc/developer-guide.md#rules). Impact options include `minor`, moderate, serious, or `critical`.

I think it would be nice to be able to configure what impact level to use for determining if a result should fail

One way to consider integrating this would be to use an [array option in yargs](http://yargs.js.org/docs/#api-reference-arraykey) with something like `--failing-impact serious critical` which would then not fail the test if `minor `or `moderate` issue occurred

**Comment by ahuth** [comment]
Great idea! I think you're right - we'll want a CLI option for this.

Could see it as either an option where we specify the levels to fail at

```
axe-storybook --failing-impact-levels serious,critical
```

or where we specify the minimum failing level (and all higher levels will also fail)

```
axe-storybook --failing-impact-level serious
```

Not sure which one makes more sense. @eoinobrien , do you want to take this on? If not, I can add it to my list.

**Comment by eoinobrien** [comment]
I have no problem taking a look at it, will try get to it during the week.

Re the flag, I think maybe the second makes more sense, I don't see a situation were someone might want serious but ignore a critical issue

**Comment by ahuth** [comment]
👍 Works for me.

If you end up getting too busy for this, no worries and let me know. Thanks!

**Comment by ahuth** [comment]
Resolved by #16
